test/api/services/package.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api
+
+import api.models.errors.ErrorWrapper
+import api.models.outcomes.ResponseWrapper
+
+package object services {
+  type ServiceOutcome[A] = Either[ErrorWrapper, ResponseWrapper[A]]
+}
```

Comment:
This package file is not required. ServiceOutcome can be imported from v1.services package file. You just need to make ServiceOutcome public instead of private in that file.